File: python/simple/terminal.py
#!/usr/bin/env python

# Simple input terminal
# MZvk 2018

def termInput():
  cli = []
  print "\033[31mPaste input, ^D to end:\033[0m"
  while True:
    try:
      input = raw_input()
      if input:
        cli.append(input)
    except EOFError:
      return cli

print termInput()

File: perl/simple/terminal.pl
#!/usr/bin/env perl

use v5.10;
use warnings;
use strict;
use Data::Dumper;

## Simple "input" terminal
## Mzvk 2019

my @data;
print "\033[31mPaste input, ^D to end:\033[0m\n";
while(my $picks = <STDIN>) {
  chomp $picks;
  push @data, $picks;
}

say +Dumper \@data;
